lib/std/compress/zstandard.zig

```diff
@@ -0,0 +1,274 @@
+const std = @import("std");
+const Allocator = std.mem.Allocator;
+
+const types = @import("zstandard/types.zig");
+
+const RingBuffer = @import("zstandard/RingBuffer.zig");
+pub const decompress = @import("zstandard/decompress.zig");
+pub usingnamespace @import("zstandard/types.zig");
+
+pub fn ZstandardStream(
```

Comment:
Please consider the fully qualified name of declarations. This one is `std.compress.zstandard.ZstandardStream`. This has a redundant "zstandard" in it. Same with the other public function in this file.

Reply:
I have removed the zstandard/Zstandard prefix, so now the streaming interface is `std.compress.zstandard.Stream`/`std.compress.zstandard.stream`.